salt/default/init.sls

```
@@ -1,7 +1,9 @@
 include:
   - default.locale
   - default.minimal
+  {% if not grains.get('no_install') | default(false) %}
```

What exactly do we want to prevent to install here, and what is the reason for that?

---

The idea is that we want to test the SUSE Manager image we produced for the public clouds as-is. If we install packages we potentially modify the installation and may mask errors in the image build.